ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/AbstractSendTransaction.java

```diff
@@ -31,16 +33,18 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-public class AbstractSendTransaction {
+public abstract class AbstractSendTransaction extends PrivacyApiMethod {
```

Comment:
nit; could this class be renamed, having an abstract class with Abstract also in the name seem redundant.
I'd suggest `PrivacySendTransaction`, but perhaps that is already taken?